### PR TITLE
Add author metadata for twifkak.

### PR DESCRIPTION
**Pull request by twifkak**
Addresses #2566.

### CODE REVIEW

**Comment by twifkak** [comment]
Build failure in deploy log seems unrelated; please advise.

**Comment by twifkak** [comment]
OK, an empty commit fixed it. Filed a bug for the CI flake.

File: site/_data/authorsData.json [hunk 1]
```diff
@@ -647,5 +647,10 @@
     "country": "US",
     "twitter": "bgalbs",
     "image": "image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg"
+  },
+  "twifkak": {
+    "twitter": "twifkak",
+    "github": "twifkak",
+    "image": "image/rULxC7pPw3PFS4o9xr7v8isFmCv1/fDaDWpuKLKDm4mjnt0jV.jpg"
```
Comment (malchata) [inline site/_data/authorsData.json:654]:
```suggestion
    "image": "image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/cshsSdVAseOneqg2APon.jpg"
```

Comment (malchata) [reply site/_data/authorsData.json:654]: Per the [images and video guidelines](https://web.dev/handbook/markup-media/), author avatars should be 384x384.

**Review by malchata — APPROVED** [review]
LGTM, just a minor nit around the author avatar dimensions.

File: site/_data/authorsData.json (hunk 1 above)
Comment (jeffposnick) [reply site/_data/authorsData.json:654]: As per a separate conversation, I think we're good using a higher resolution image URL, and allowing the image CDN to downsize it as needed.

Comment (malchata) [reply site/_data/authorsData.json:654]: +1. @twifkak, feel free to reject this change, as my interpretation of the images and video guidelines may have been too strict.

Comment (twifkak) [reply site/_data/authorsData.json:656]: OK, will reject. Yeah, I uploaded the high resolution per the recommendation atop the [Choose a file](https://web.dev/handbook/markup-media/#choose-a-file) section of that doc:

> "Upload a high quality image... Our image CDN... will resize the image so you don't have to."

**Comment by twifkak** [comment]
Thanks!

**Pull request merged**